apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -185,4 +212,9 @@ var AdvancedShareOptions = Radium(React.createClass({
   }
 }));
 
-module.exports = AdvancedShareOptions;
+export default connect(
+  state => ({
+    embedUrl: state.pageConstants.iframeEmbedUrl,
+    noSourceEmbedUrl: state.pageConstants.iframeNoSourceEmbedUrl,
+  })
+)(AdvancedShareOptions);
```

Comment:
The new props from connecting look unused.  Did we need to connect this component at all?